redhat-director-scripts/prepare_artifacts.sh

```diff
@@ -31,19 +31,21 @@ tvault_version=`curl -s http://${tvault_ip}:8081/packages/ | grep tvault-contego
 tvault_release=`echo $tvault_version | awk '{split($0,a,"."); print a[1], a[2]}'`
 tvault_release=`echo $tvault_release | sed 's/\ /\./'`
 
-rpm1="puppet-triliovault-${tvault_version}-${tvault_release}.noarch.rpm"
-
-curl -O http://${tvault_ip}:8085/yum-repo/${rpm1}
-
-if [ ! -f $rpm1 ]; then
-    echo "rpm download failed, rpm name: $rpm1"
-    exit 1
-fi
-
+##Create template to upload trilio.repo to overcloud nodes
 rm -rf etc/
 rm -f triliorepo.tgz
 mkdir -p ${basedir}/etc/yum.repos.d/
 cp ${basedir}/trilio.repo.template ${basedir}/etc/yum.repos.d/trilio.repo
 sed -i "s/TVAULTIP/${tvault_ip}/" ${basedir}/etc/yum.repos.d/trilio.repo
```

Comment:
Instead of Trilio IP, gemfury can be used. This is just an alternative. 
Code changes look good.

Reply:
This is for 3.0. In 3.2 we have used our public yum repo server. Gemfury does not hold our rpms.